MAUI/Cartesian-Charts/StackedArea100.md

```diff
@@ -100,4 +100,104 @@ this.Content = chart;
+                                 ShowMarkers="True"/>
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+...
+StackingArea100Series series = new StackingArea100Series()
+{
+    ItemsSource = new ViewModel().StackData,
+    XBindingPath = "Year",
+    YBindingPath = "Value",
+    ShowMarkers= true
+};
+...
+chart.Series.Add(series);
+this.Content= chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Marker customization
+
+In order to change the series markers’ appearance, create an instance of the [MarkerSettings]() property. The following properties are used to customize marker appearance.
+
+* [Type](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Type), of type `ShapeType`, describes the shape of the series marker. The default value of this property is the [ShapeType.Circle]().
+* [Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Stroke), of type `Brush`, indicates the brush used to paint the marker border.
+* [StrokeWidth](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_StrokeWidth), of type `double`, indicates the width of the marker border.
+* [Fill](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Fill), of type `Brush`, indicates the color of the marker.
+* [Width](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Width), of type `double`, indicates the width of the marker.
+* [Height](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.ChartMarkerSettings.html#Syncfusion_Maui_Charts_ChartMarkerSettings_Height), of type `double`, indicates the height of the marker.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+    ...
+    <chart:StackingArea100Series ItemsSource="{Binding StackData}"
+                                 XBindingPath="Year"
+                                 YBindingPath="Value"
+                                 ShowMarkers="True">
+        <chart:StackingArea100Series.MarkerSettings>
+            <chart:ChartMarkerSettings Type="Diamond"
+                                       Fill="LightBlue"
+                                       Stroke="Blue"
+                                       StrokeWidth="1"
+                                       Height="8"
+                                       Width="8"/>
+        </chart:StackingArea100Series.MarkerSettings>
+    </chart:StackingArea100Series>
+    ...
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+...
+ChartMarkerSettings chartMarker= new ChartMarkerSettings();
```

Comment:
set chartMarker instance to chart marker setting

Reference -https://help.syncfusion.com/maui/cartesian-charts/line#marker-customization
<img width="569" alt="image" src="https://github.com/user-attachments/assets/67a6df4c-ea49-4537-9d26-b851630c25a8">

Reply:
Set the chartMarker instance to MarkerSettings for the necessary charts.
Modified the mentioned changes under migration section for all the five charts.